docs/en/stack/ml/get-started/ml-getting-started.asciidoc

```diff
@@ -1,6 +1,53 @@
 [[ml-getting-started]]
 == Getting started with {ml}
 
+{ml-cap} features analyze your data and generate models for its patterns of
+behavior. The type of analysis that you choose depends on the questions or
+problems you want to address and the type of data you have available.
+
+[discrete]
+[[get-started-unsupervised]]
+=== Unsupervised {ml}
+
+There two types of analysis that can deduce the patterns and relationships
+within your data without training or intervention: _{anomaly-detect}_ and
+_{oldetection}_.
+
+<<xpack-ml,{anomaly-detect-cap}>> requires time series data. It constructs a
+probability model and can run continuously to identify unusual events as they
+occur. The model evolves over time; you can use its insights to forecast future
+anomalous events.
```

Comment:
As far as I know, forecast doesn't predict anomalous events, it predicts what's the expected behavior based on the patterns that the model saw so far.